app/models/post.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# :nodoc:
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :likes, dependent: :destroy
+  default_scope { order(created_at: :desc) }
+  validates :description, presence: true
+  has_many :comments
```

Comment:
If we destroy a post, shouldn't all of its associated models (including comments) be destroyed as well? I suggest adding `dependent: :destroy` here as well.

[`OPTIONAL`]